Contributor.md

```diff
@@ -0,0 +1,88 @@
+
+# Contributor Guidelines
+
+Thank you for considering contributing to our project! We welcome contributions of all kinds, including bug fixes, new features, documentation, and tests. Please follow the guidelines below to ensure a smooth contribution process.
+
+## How to Contribute
+
+### 1. Fork the Repository
+        Fork this repository to your GitHub account and clone it locally.
+
+        ``` 
+        git clone https://github.com/your-username/project-name.git
+        cd project-name 
+        ```
+
+### 2. Fork the Repository
+        Follow the project’s setup instructions in the `README.md` file to prepare your development environment.
+
+### 3. Create a Branch
+        Create a new branch for your contribution. Use a descriptive name.
+
+        ``` 
+        git checkout -b feature/your-feature-name 
+        ```
+
+### 4. Make Your Changes
+        Implement your changes, and ensure they adhere to the project's coding standards.
+
+### 5. Write Tests
+        Add or update tests to cover your changes, if applicable.
+
+### 6. Run Tests
+        Run all tests to ensure your changes do not break existing functionality.
+
+        ``` 
+        make test 
+        ```
+
+### 7. Commit Your Changes
+        Write a clear and concise commit message.
+
+        ``` 
+        git add .
+        git commit -m "Add feature: your-feature-name" 
+        ```
+
+### 8. Push Your Changes
+        Push your branch to your forked repository.
+
+        ``` 
+        git push origin feature/your-feature-name 
+        ```
+
+### 9. Push Your Changes
+        Push your branch to your forked repository.
+
+        ``` 
+        git push origin feature/your-feature-name 
+        ```
+
+### 10. Create a Pull Request
+        Open a pull request (PR) to the main repository. Include a detailed description of your changes and any related issue numbers.
+
+## Code of Conduct
+        Please read and adhere to our [Code of Conduct](https://github.com/veraison/corim/blob/main/CODE_OF_CONDUCT.md). Respectful and inclusive behavior is expected from all contributors.
+
+## Guidelines for Contributions
```

Comment:
This section `Guidelines for Contributions` should now be re-worked/moved and aligned with your top level Contributor Guidelines for smooth logical flow!